config/samples/full_config.yaml

```diff
@@ -193,77 +224,402 @@ spec:
+          default-otlp:
+            name: logs-otlp
+            config-name: endpoint-logs-otlp
+            properties:
+              content_type: Otlp
         events:
           default:
             name: events
             config-name: endpoint-events
+            properties:
+              default_date_formats:
+                ## Ensures that timestamp key has precedence over timestamp auto discovery
+                - format: epoch
+                  locator: '\"timestamp\":(\\d+)'
+          default-otlp:
+            name: events-otlp
+            config-name: endpoint-events-otlp
+            properties:
+              content_type: Otlp
         traces:
           default:
             name: traces
             config-name: endpoint-traces
             properties:
               content_type: Zipkin
+          default-otlp:
+            name: traces-otlp
+            config-name: endpoint-traces-otlp
+            properties:
+              content_type: Otlp
+
+    ### Global configuration for OpenTelemetry Collector
+    otelcolImage:
+      repository: "public.ecr.aws/sumologic/sumologic-otel-collector"
+      tag: "0.102.1-sumo-0"
+
+      ## Add a -fips suffix to all image tags. With default tags, this results in FIPS-compliant otel images.
+      ## See https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/fips.md for more information.
+      addFipsSuffix: false
 
     ### Configuration for collection of Kubernetes events
     events:
-      provider: fluentd
+      enabled: true
+
+      ## Source name for the Events source. Default: "events"
+      sourceName: "events"
+
+      ## Source category for the Events source. Default: "" which is resolved to "{clusterName}/events"
+      # sourceCategory: "kubernetes/events"
+
+      ## Used to replace '-' with another character.
+      sourceCategoryReplaceDash: "/"
+
+      persistence:
+        enabled: true
+        size: 10Gi
+
+        ## Configuration for the Persistent Volume and Persistent Volume Claim
+        ## where the storage is kept
+        persistentVolume:
+          path: /var/lib/storage/events
+          accessMode: ReadWriteOnce
+          ## Add custom labels to otelcol event statefulset PVC
+          pvcLabels: {}
+          # storageClass:
+
+      sourceType: otlp
 
     ### Logs configuration
     ## Set the enabled flag to false for disabling logs ingestion altogether.
     logs:
       enabled: true
-      metadata:
-        ## Set provider service (either fluentd or otelcol).
-        provider: fluentd
 
       collector:
         otelcol:
+          enabled: true
+        ## Experimental
+        otellogswindows:
           enabled: false
-
-        ## Allow running otel and Fluent-Bit side by side. This will result in duplicated
-        ## logs being ingested. Only enabled this if you're **certain** it's what you want.
-        allowSideBySide: false
+        otelcloudwatch:
+          enabled: false
```

Comment:
We could skip it on Openshift, but AFAIU it will be easier to maintain this file by having it as close as possible to the chart one?

Reply:
This file contains all the settings from values.yaml intentionally, other configuration files are adjusted to OpenShift.
It documented here: https://github.com/SumoLogic/sumologic-kubernetes-collection-helm-operator/tree/main/config/samples